stable/rabbitmq/values.yaml

```diff
@@ -77,6 +71,7 @@ rabbitmq:
 
   ## Configution file content
   configuration: |-
+      disk_free_limit.absolute = 4GB
```

Comment:
Do we want to use this value as a default? Maybe keep the rabbitmq default? Maybe keep it as an example in `values-production.yaml`? Or commented-out here?

In fact the `configuration` value should ideally be split to allow the chart users to extend it without overriding it (as it contains many different configs which could be changed independently of one another).

Reply:
So you are suggesting to have a `configuration` and `extendedConfiguration` for example?
So each entry in `extendedConfiguration` is appended to the `configuration` 

Regarding the default value for `disk_free_limit.absolute` I agree on setting the default value (50MB)

Reply:
> So you are suggesting to have a `configuration` and `extendedConfiguration` for example?
> So each entry in `extendedConfiguration` is appended to the `configuration`

I'm not sure about the naming, but part of the configuration is required for the statefulset to work (k8s clustering formation on rabbitmq), and should probably be kept untouched by default even when the chart user wants to define (unrelated) custom configuration. (not sure about `loopback_users.guest` and your proposed `disk_free_limit.absolute` though..)

So the idea could be to move the required config to `clusterConfiguration`, and let `configuration` empty and ready to be set by the chart user. Or maybe if we want to include `loopback_users.guest` too, maybe call it `baseConfiguration` ? Or keep it as `configuration` and ask users to use `extraConfiguration` or `extendedConfiguration` as you suggested, but the risk is that charts users would accidentally overwrite the simply named `configuration` instead of the new one.


> 
> Regarding the default value for `disk_free_limit.absolute` I agree on setting the default value (50MB)

OK, then maybe simply *not* set it in configuration and let rabbitmq use its default value ?

Reply:
So I guess we can simply move all the cluster configuration directly to the configMap and the `configuration` parameter from the values will be available for users to add custom configuration that will be appended.

Reply:
This would make it impossible for the chart users to replace the cluster configuration, which in some cases would still be useful I think. That's why I proposed two values.

Reply:
That is true. I wasn't sure if replacing the cluster configuration was useful in some case but I think we should allow it just in case. Thanks